int64 stamp
float64[] currentPos
float64[] targetPos
float64[] newTargetPos
float64[] errorValue
float64[] vs
float64[] vel
float64[] maxVelocity
float64[] effort
float64[] simTime
float64[] boxPosition
float64[] boxOrientation
